src/app/[locale]/topics/[topicId]/(topic)/components/mindmap/MindmapContent.tsx

```
@@ -0,0 +1,68 @@
+import { useTheme } from 'next-themes';
+import React from 'react';
+
+//set react flow to use custom nodes & edges
+const nodeTypes = {
+    'custom-react-flow-node': CustomReactFlowNode,
+};
+const edgeTypes = {
+    floating: FloatingEdge,
+};
+
+const MindmapContent = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)